src/Groups/matrices/MatGrp.jl

```diff
@@ -148,14 +148,7 @@ function Base.show(io::IO, x::MatrixGroup)
    end
 end
 
-function Base.show(io::IO, x::MatrixGroupElem)
-   if isdefined(x, :elm)
-      show(io, "text/plain", x.elm)
-#      print(io, x.elm)
-   else
-      print(io, String(GAP.Globals.StringViewObj(x.X)))
-   end
-end
+Base.show(io::IO, x::MatrixGroupElem) = show(io, "text/plain", x.elm)
```

Comment:
@tthsqe12 Why not pass through `expressify`?

Reply:
It will pass through expressify eventually. What are you trying to get at?

Reply:
Matrix printing doesn't work like that because show via expressify is incapable of producing 2D text output. Do you remember when we agreed to scrap the general 2D printer because no one wants to see polynomials or fractions in 2D - just matrices?

Reply:
ok

Reply:
So is it ok if I include Dan's proposal?
(My aim was to improve *what* is shown, I had not intended to deal with the formatting.)

Reply:
Yes, what I suggested does not help. Just use Dan's proposal.